### ISSUE TITLE
Theme support request

### ISSUE THREAD

**Issue by mariosant** [agorf/lucien#1]
Thank you for creating lucien!

Would you please consider support theming? The UI is great as it is now, but it might be a bit too whitey for a lot of people's tastes (I am one of those).

**Comment by agorf** [comment]
Yep, that's actually a good idea. How about making it possible to load a custom CSS file?

Styling the textarea is trivial (background and text color) but styling the HTML area would take some effort I imagine.

**Comment by mariosant** [comment]
A custom css file would be great, @agorf! Don't worry about the effort, the internet is full of themes which one could convert and use. I am happy to contribute a few, once you decide how to handle this.

**Comment by agorf** [comment]
It would not be that easy. You'd have to copy and set t he colors for this stylesheet: https://github.com/sindresorhus/github-markdown-css/blob/gh-pages/github-markdown.css

It has 59 color properties. :cold_sweat:

**Comment by mariosant** [comment]
Ouch! Maybe [css variables](https://www.w3schools.com/css/css3_variables.asp) can be of help there.

**Comment by agorf** [comment]
As Lucien currently does not use SASS/SCSS, this is not an option. Moreover, you'd still have to go to 59 points to extract them to variables.

This needs a bit of thought. I think one intermediate step would be to provide an option for a built-in dark theme with some sane colors (like many sites and Android apps to with "dark mode"). It will not be customizeable, but you'd at least have a dark option.

**Comment by mariosant** [comment]
Sure, @agorf! Just to clarify, css variables are part of standard CSS. Check out https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Having light or dark mode would be great as well!

**Comment by agorf** [comment]
Cool, didn't know that!
